userguide/system.rst

```diff
@@ -822,6 +818,108 @@ with locking enabled:
        LockOnReset:     1
 
 
+.. index:: SED, SED Password
+.. _Managing SED Password and Data:
+
+Managing SED Password and Data
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This section contains command line instructions to manage SED
+passwords and data. The command used is
+`sedutil-cli(8) <https://www.mankier.com/8/sedutil-cli>`__. Most
+SEDs are TCG-E (Enterprise) or TCG-Opal
+(`Opal v2.0 <https://trustedcomputinggroup.org/wp-content/uploads/TCG_Storage-Opal_SSC_v2.01_rev1.00.pdf>`__).
+
+.. warning:: These commands can be descructive to data and passwords.
+   Take necessary data backup procedures and use the commands with
+   caution.
+
+Drives can be checked individually to determine if it is a SED and what
```

Comment:
Too many things in one sentence.  Split out the part about individual or batch:
`Drive SED version can be checked on a single drive with this command:`
`All connected disks (just SED?) can be checked at once:`